source/_components/device_tracker.bt_smarthub.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: "BT Smart Hub"
+description: "Instructions on how to integrate BT Smart Hub router into Home Assistant."
+date: 2018-10-05 10:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: bt.png
+ha_category: Presence Detection
+ha_release: "0.79.4"
```

Comment:
New features are never released in a minor release. So please set it to the next major release "0.80"

Reply:
Thanks